### ISSUE TITLE
Action bug - switch to prev/next in folder.

### ISSUE THREAD

**Issue by nounaim** [Jeff0S/sws#753]
Actions dont work if item is reversed, please fix this problem, thanks.

**Comment by Jeff0S** [comment]
Yeah, I know what that prob is. Many Xenakios take-related actions are buggy ans it's a bit like abandon-ware..
I'm tempted to A) disable all his actions that have been reported as buggy (fixing those would require too much time for me) B) until some kind contributor will take care of them.

A probably needs B to happen. Thinking about it...
